### PR TITLE
WEL rule probe and separate test "site" 

### PR DESCRIPTION
**Pull request by TrevorFSmith**
I created a [WEL test probe](https://github.com/fullstorydev/web-embed-lab/blob/master/docs/TEST_PROBE_API.md) and a testing page that work together to ensure that the rules ran, that the right number of results were created, and that the results have expected parameters.

In the future we can flesh this out to test the intricacies of each operator but this satisfies the task which is to make sure that the DLO successfully evaluates rules.

### CODE REVIEW

File: test/web-embed-lab/test-probes/dlo-rules/test.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
+/**
+DLORulesProbe tests that a well known array contains expected rule results
+*/
+class DLORulesProbe {
+  /**
+  @return {Object} data collected when the target embed script *is not* loaded
+  @return {Object.success} always true
+  */
+  async gatherBaselineData(){
+    console.log('DLO rules baseline')
```
Comment (van-fs) [inline test/web-embed-lab/test-probes/dlo-rules/test.js:10]: nit: semicolon missing.  I suspect our linter isn't working on these tests.

File: test/web-embed-lab/test-probes/dlo-rules/test.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+          results.failures.push('The parameters array must contain arrays like [index, value]')
+        }
+      }
+    }
+
+    return results
+  }
+}
+
+window.__welProbes['dlo-rules'] = new DLORulesProbe()
```
Comment (van-fs) [inline test/web-embed-lab/test-probes/dlo-rules/test.js:59]: same nit; semicolon.

**Review by van-fs — APPROVED** [review]
LGTM; two super-nits that have no functional impact.

File: test/web-embed-lab/test-probes/dlo-rules/test.js (hunk 1 above)
Comment (TrevorFSmith) [reply test/web-embed-lab/test-probes/dlo-rules/test.js:10]: Yep, the WEL isn't using the linter (or semicolons) so the new test is consistent with the other test probes from the WEL.

**Pull request merged**